FTBFS in src/ptrace/_UPT_ptrauth_insn_mask.c on x86_64

**Environment**
- libunwind version: master
- architecture: x86_64
- OS: Ubuntu 20.04

**Description**

I get a failure to build from source building for x86_64-pc-linux-gnu on Ubuntu
20.04 due to invalid C code.
```
../../src/ptrace/_UPT_ptrauth_insn_mask.c: In function ‘_UPT_ptrauth_insn_mask’:
../../src/ptrace/_UPT_ptrauth_insn_mask.c:52:36: error: parameter name omitted
   52 | unw_word_t _UPT_ptrauth_insn_mask (unw_addr_space_t, void *)
         |                                    ^~~~~~~~~~~~~~~~
         ../../src/ptrace/_UPT_ptrauth_insn_mask.c:52:54: error: parameter name
         omitted
            52 | unw_word_t _UPT_ptrauth_insn_mask (unw_addr_space_t, void *)
                  |                                                      ^~~~~~
                  make[2]: *** [Makefile:4868: ptrace/_UPT_ptrauth_insn_mask.lo]
                  Error 1
                  make[1]: *** [Makefile:2967: all] Error 2
                  make: *** [Makefile:623: all-recursive] Error 1
```

Note that for whatever reason this does not happen in CI.

**Reproduction Steps**
Steps to reproduce the behavior:
1. Checkout out clean source
2. `autoreconf -if && ./configure && make`
3. See error

--- a/src/ptrace/_UPT_ptrauth_insn_mask.c
+++ b/src/ptrace/_UPT_ptrauth_insn_mask.c
@@ -49,9 +49,10 @@ unw_word_t _UPT_ptrauth_insn_mask (UNUSED unw_addr_space_t as, void *arg)
 
 #else
 
-unw_word_t _UPT_ptrauth_insn_mask (unw_addr_space_t, void *)
+unw_word_t _UPT_ptrauth_insn_mask (UNUSED unw_addr_space_t as, UNUSED void *arg)
 {
   return 0;
 }
 
 #endif
+

